Carmunication/Controllers/SessionList/GroupAdd/SelectStartTimeViewController.swift

```diff
@@ -40,8 +45,17 @@ extension SelectStartTimeViewController {
     }
 
     @objc private func saveButtonAction() {
+        if let selectedTime = selectedTime { // 선택된 시간을 클로저에 전
+            timeSelectionHandler?(selectedTime)
+        } else { // 바로 저장을 눌렀다면 현재시간을 주입
+            timeSelectionHandler?(Date())
+        }
```

Comment:
이 코드는 좀 더 간단하게 작성할 수도 있을 것 같아요

```suggestion
        timeSelectionHandler?(selectedTime ?? Date()
```

Reply:
클로저가 두 번 쓰이는 곳이 있는데, 거기에서는 이렇게 쓰고, 여기에서는 제안해주신 대로 반영하는 것이 맞는것 같습니다 수정하도록 하겠습니다!